addons/auth_ldap/__manifest__.py

```diff
@@ -11,6 +11,6 @@
         'views/res_config_settings_views.xml',
     ],
     'external_dependencies' : {
-        'python' : ['ldap'],
+        'python' : ['pyldap'],
```

Comment:
shouldn't this be `python-ldap` now?

Reply:
I think so.

Reply:
@mart-e @KangOl in requirements.txt it's pyldap, so it should be changed there too I guess? Unless there is a reason to stay with pyldap?

Reply:
I guess the requirements is historical as both have been merged. Before your change, as long as I could do an `import ldap`, the package providing it did not matter. Now I am not able anymore to install `auth_ldap` as it is provided by `python-ldap` on my machine.
Maybe this is a good example why the backward compatibility is needed :)